ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -29,6 +29,7 @@ FCU_DFSS_CSP_CHWDC:
   - CSP
   - CHWDC
 
+  
```

Comment:
If this is just whitespace diff between PRs, please revert.

Reply:
This is just a white space....